spec/ruby/shared/types/rb_num2dbl_fails.rb

```diff
@@ -0,0 +1,19 @@
+#
+# Shared tests for rb_num2dbl related conversion failures.
+#
+# Usage example:
+#   it_behaves_like :rb_num2dbl_fails, nil, -> v { o = MyObject.new; -> { o.blocking_io(timeout: v) } }
+#
+
+describe :rb_num2dbl_fails, shared: true do
+  ruby_version_is "3.2" do
```

Comment:
**minor:** Do we need this version guard.  I suppose that this coercion rules might have been introduced in CRuby long time ago.

Reply:
Ah forgot this. Thanks.